examples/with-shell-commands/turbo.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "dependsOn": ["prebuild", "^build"],
+      "outputs": ["output-file.txt", "dist/**"]
+    },
+    "prebuild": {},
+    "lint": {},
+    "type-check": {}
```

Comment:
Maybe an addition of an `build-error` task (`echo 'build failed' && exit 1`) on a package? Something I find myself doing for testing a good amount.

Reply:
My gut is that that would make it more difficult for general use. Let's see if we end up with a need for that in the wild. 👍